cibuildwheel/macos.py

```diff
@@ -130,7 +162,7 @@ def call(args, env=None, cwd=None, shell=False):
         if test_command:
             # set up a virtual environment to install and test from, to make sure
             # there are no dependencies that were pulled in at build time.
-            call(['pip', 'install', 'virtualenv'], env=env)
+            call(['pip', 'install', 'virtualenv<20'], env=env)
```

Comment:
Here and in circle.ci configuration. Why?

Reply:
Because these are two separate issues.